controllers/apply/lib/form-model.js

```diff
@@ -137,10 +140,22 @@ class FormModel {
                 }
             }
 
+            function sectionStatusLabel() {
```

Comment:
Added this here so avoid the logic in the templates, making it easier to work with progress in macros.